modules/core/test/test_rotatedrect.cpp

```diff
@@ -105,4 +105,82 @@ int Core_RotatedRectConstructorTest::validate_test_results( int )
 
 TEST(Core_RotatedRect, three_point_constructor) { Core_RotatedRectConstructorTest test; test.safe_run(); }
 
+TEST(Core_RotatedRect, intersection)
```

Comment:
This test should be moved to `imgproc` module, see `test_intersection.cpp` file

Reply:
:+1: